### PR TITLE
Add os_user_facts module

### PR DESCRIPTION
**Pull request by rcarrillocruz**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME

os_user_facts
##### ANSIBLE VERSION

```
ansible 2.1.0 (devel cca084c89d) last updated 2016/04/05 14:55:00 (GMT +000)
  lib/ansible/modules/core: (detached HEAD cf01087a30) last updated 2016/04/05 14:55:07 (GMT +000)
  lib/ansible/modules/extras: (os_user_facts f2d4830467) last updated 2016/04/06 10:22:37 (GMT +000)
  config file = 
  configured module search path = Default w/o overrides
```
##### SUMMARY

This module gather facts about one or more OpenStack users

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @rcarrillocruz. This module is going into community review.

We notice that you are one of the maintainers (@emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/openstack/os_user_facts.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+extends_documentation_fragment: openstack
+version_added: "2.1"
+author: "Ricardo Carrillo Cruz (@rcarrillocruz)"
+description:
+    - Retrieve facts about a one or more OpenStack users
+requirements:
+    - "python >= 2.6"
+    - "shade"
+options:
+   user:
```
Comment (Shrews) [inline cloud/openstack/os_user_facts.py]: This does not match parameter name, which is 'name'.

File: cloud/openstack/os_user_facts.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    contains:
+        id:
+            description: Unique UUID.
+            returned: success
+            type: string
+        name:
+            description: Name given to the user.
+            returned: success
+            type: string
+        description:
```
Comment (Shrews) [inline cloud/openstack/os_user_facts.py]: This field, description, doesn't appear in devstack output.  ????

File: cloud/openstack/os_user_facts.py [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+            returned: success
+            type: string
+        default_project_id:
+            description: Default project ID of the user
+            returned: success
+            type: string
+        email:
+            description: Email of the user
+            returned: success
+            type: string
```
Comment (Shrews) [inline cloud/openstack/os_user_facts.py:114]: 'username' is also a return value.

File: cloud/openstack/os_user_facts.py (hunk 1 above)
Comment (rcarrillocruz) [inline cloud/openstack/os_user_facts.py]: done

File: cloud/openstack/os_user_facts.py (hunk 2 above)
Comment (rcarrillocruz) [inline cloud/openstack/os_user_facts.py]: true, removing, must have been a copy-pasta from other module leftover

File: cloud/openstack/os_user_facts.py (hunk 3 above)
Comment (rcarrillocruz) [inline cloud/openstack/os_user_facts.py:114]: you are right, although I'm not sure why it is returned and what is it about.

In the API docs is not listed as response parameter:

http://developer.openstack.org/api-ref-identity-v3.html#users-v3

I will put it on the return string, but will leave it as 'username of the user', don't really know what is supposed to be this value and if this is  a bug on keystone client or maybe the docs?

**Comment by rcarrillocruz** [comment]
ready_for_review

**Comment by gregdek** [comment]
Thanks @rcarrillocruz. To the current maintainers, @emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit' or 'needs_revision' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Pull request merged**